src/components/Navbar/NavTabs.js

```diff
@@ -33,64 +34,74 @@ import "./NavAccessibility.css"
       return () => window.removeEventListener("resize", handleResize);
     }, []); // Empty array ensures that effect is only run on mount
 
-    // The padding value as a function of screen size
-    let navbarPadding,navbarHeight, navitemWidth, logoWidth, logoFontSize; 
-    let buttonHeight, buttonWidth, heartFontSize, heartWidth;
-    let navtabsDisplay, burgerMenuDisplay;
+    // Navbar and childs slyle properties
+    let navbarSideMargin, navbarHeight, navtabsDisplay,navitemWidth;
+    let logoWidth, logoFontSize, logoTextMaxWidth; 
+    let buttonHeight, buttonWidth, buttonFontSize, heartWidth;
+    let burgerMenuDisplay, burMenuButtonPosition;
     
     if (windowSize.width >= 1040) {
-      navbarPadding = (windowSize.width / 4 - 240).toFixed(1);
+      navbarSideMargin = (windowSize.width / 4 - 240).toFixed(1);
       navbarHeight = 80;
       navitemWidth = ((windowSize.width - 720) / 5.5).toFixed(1);
       logoFontSize = 20;
       logoWidth = 48;
+      logoTextMaxWidth = 250;
       buttonHeight = 48;
       buttonWidth = 156;
       navtabsDisplay = "inline-block"
       burgerMenuDisplay = "none"
-      heartFontSize = 18;
+      burMenuButtonPosition = "55%"
+      buttonFontSize = 18;
       heartWidth=17
     } else if (windowSize.width >400 && windowSize.width < 1040) {
-      navbarPadding = (windowSize.width / 45 + 7).toFixed(1);
+      navbarSideMargin = (windowSize.width / 45 + 7).toFixed(1);
       navbarHeight = 58;
       logoWidth = 40;
       logoFontSize = 16;
+      logoTextMaxWidth = (0.2*windowSize.width + 50);
       buttonHeight = 40;
-      buttonWidth = (windowSize.width *0.11).toFixed(0);
+      buttonWidth = 105;
       navtabsDisplay = "none";
-      burgerMenuDisplay = "inline-block";
-      heartFontSize = 14;
+      burgerMenuDisplay = "block";
+      burMenuButtonPosition = "50%"
+      buttonFontSize = 14;
       heartWidth=15;
     } else if(windowSize.width <=400){
-      navbarPadding = 14; // small screen
+      navbarSideMargin = 14; // small screen
       navbarHeight = 48;
+      navitemWidth = 105;
       logoWidth = 30;
       logoFontSize = 12;
+      logoTextMaxWidth = 0.21*windowSize.width+10;
       buttonHeight = 32;
       buttonWidth = 96;
       navtabsDisplay = "none";
-      burgerMenuDisplay = "inline-block";
-      heartFontSize = 12;
+      burgerMenuDisplay = "block";
+      burMenuButtonPosition = "47%"
```

Comment:
**REVIEW**: burMenuButtonPosition should be such that it aligns to the left

Reply:
My bad. Did not go back on figma. Please handle that part to complete the side bar.

Reply:
@BBoutet1 u r right, figma is so annoying bcos so heavy!! we should request ux team to split figma into separate artefacts rather than one monolith